mlir/utils/jenkins/Jenkinsfile.downstream

```diff
@@ -41,8 +43,27 @@ pipeline {
                         -DMLIR_ROCM_RUNNER_ENABLED=1 \
                         -DMLIR_MIOPEN_DRIVER_ENABLED=1
 
-                    # build LLVM / MLR and run tests
-                    cmake --build . --target check-mlir
+	     	    # premerge checks
+                    cd ..
+                    if [ ! -f ./compile_commands.json ]; then
+                       ln -s build/compile_commands.json compile_commands.json
+                    fi
```

Comment:
Can we get rid of this `cd..`? The `ln -s` can handle a different path.

Reply:
No. I cannot invoke the static checks in the build directory. That would cause errors. I'm exactly sure why.

Reply:
Could you add a comment saying that the static checks must be carried out in the root directory?